src/library/scala/collection/immutable/Queue.scala

```diff
@@ -84,6 +84,14 @@ sealed class Queue[+A] protected(protected val in: List[A], protected val out: L
     else if (in.nonEmpty) new Queue(Nil, in.reverse.tail)
     else throw new NoSuchElementException("tail on empty queue")
 
+  /*This is made to avoid inefficient implementation of iterator*/
```

Comment:
Please delimit `/*` and `This` with a space, and change to `iterator. */` (dot and a space).
We should have lint tests for this in the future, but for now we just ensure it during review.